vignettes/calibration-2021.Rmd

```diff
@@ -0,0 +1,292 @@
+---
+title: "Winter Run Calibration 2021"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Winter Run Calibration 2021}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+library(tidyverse)
+library(plotly)
+
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Objective
+
+We are using genetic algorithms, a type of stochastic optimization, to calibrate the following model coefficients:
+
+* `..surv_adult_enroute_int` used in submodel `surv_adult_enroute`
+* `..surv_juv_rear_int` used in submodel `surv_juv_rear`
+* `..surv_juv_rear_contact_points` used in submodel `surv_juv_rear`
+* `..surv_juv_rear_prop_diversions` used in submodel `surv_juv_rear`
+* `..surv_juv_rear_total_diversions` used in submodel `surv_juv_rear`
+* `..surv_juv_bypass_int` used in submodel `surv_juv_bypass`
+* `..surv_juv_delta_int` used in submodel `surv_juv_delta`
+* `..surv_juv_delta_contact_points` used in submodel `surv_juv_delta`
+* `..surv_juv_delta_total_diverted` used in submodel `surv_juv_delta`
+* `..surv_juv_outmigration_sj_int` used in submodel `surv_juv_outmigration_san_joaquin`
+* `..ocean_entry_success_int`  used in submodel `ocean_entry_success`
+
+A detailed description for each of these parameters can be found in the 
+[Winter Run Documentation Website](https://cvpia-osc.github.io/winterRunDSM/reference/params.html)
+
+We are restricting the calibration to watersheds with Grandtab data, these are:
+
+```{r, echo=FALSE}
+keep <- c(1, 3)
+knitr::kable(winterRunDSM::watershed_attributes[keep, "watershed", drop = FALSE])
+```
+
+
+## Packages
+
+```{r, message=FALSE}
+library(GA)
+library(winterRunDSM)
```

Comment:
this is generating a warning please suppress warning in the r code chunk header

Reply:
fixed